### PR TITLE
feat(adr): use defined process to propose, approve or reject ADRs

### PR DESCRIPTION
**Pull request by esciara**

### CODE REVIEW

**Comment by slequeux** [comment]
Is this an architecture decision ? I'm not sure. if it. needs an ADR

**Comment by esciara** [comment]
> Is this an architecture decision ? I'm not sure. if it. needs an ADR

@slequeux I remove the ADR. If you are happy with the process, will squash and change the commit message for clarity before rebase.

File: README.md [hunk 1]
```diff
@@ -11,21 +11,100 @@ To learn about ADR, see:
...
 
 
-## Process for proposing a new ADR
+## Proposing a new ADR
 
-1. Create a feature branch from `master`.
-1. Create a new file in `docs/adr/` using the template in place.
+Please note that:
+
+* ADR uses MADR template made available in [docs/adr/template.md](docs/adr/template.md)
```
Comment (slequeux) [inline README.md]: Proposition : `ADR uses [MADR template](docs/adr/template.md)`

File: README.md [hunk 2]
```diff
@@ -11,21 +11,100 @@ To learn about ADR, see:
...
+* Only one ADR must be proposed per feature branch (on file with `XXXX` in at the
```
Comment (slequeux) [inline README.md]: `on` => `one` ?

File: README.md [hunk 3]
```diff
@@ -11,21 +11,100 @@ To learn about ADR, see:
...
+1. Fill your ADR:
+   * Status line exists and is set to `proposal`
+   * Date line exists and is set to current date
+1. Once finish, commit your work with message either
+   (title follows [MADR-0005-format](https://github.com/adr/madr/blob/2.1.2/docs/adr/0005-use-dashes-in-filenames.md)):
+   * When working on a repo dedicated to ADRs:
+
+     ```
+     $ git commit -m 'feat(adr): <adr-title-in-lowercase>'
+     ```
```
Comment (slequeux) [inline README.md]:
```bash
git commit -m 'feat(adr): <adr-title-in-lowercase>'
```

File: README.md (hunk 1 above)
Comment (esciara) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (esciara) [reply README.md]: done everywhere.

File: README.md (hunk 2 above)
Comment (esciara) [reply README.md]: done

**Pull request merged**